packages/styles/scss/patterns/blocks/callout-with-media/index.scss

```diff
@@ -5,6 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/// @deprecated Remap import to "@carbon/ibmdotcom-styles/scss/components/callout-with-media/index"
+
+@warn 'Deprecated scss import for `callout-with-media`, remap to `@carbon/ibmdotcom-styles/scss/components/callout-with-media/index`';
 @import '../../../globals/imports';
```

Comment:
Hey Kenny, for all of these just import the final entry point (`.../.../.../components/callout-with-media/index`) so we don't have to manage in two places